scripts/entrypoint.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 set -e
 
+/scripts/entrypoint_openshift.sh
```

Comment:
CMIIW, if we use this approach then this setup is not specific to openshift only. I think a more general name can do. Such as :`passwd_setup.sh`

Reply:
valid point, thanks, will change.